client/src/pages/market-selection/page.tsx

```
@@ -0,0 +1,334 @@
+      name: 'Neighborhood Grocers',
+      address: '987 Cedar Lane, Southside',
+      distance: '2.8 miles',
+      hours: '7:00 AM - 11:00 PM',
+      rating: 4.4,
+      lat: 40.7050,
+      lng: -74.0020
+    }
+  ];
+
+  // State for API data
+  const [markets, setMarkets] = useState<Market[]>([]);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'markets'. Consider to delete it or rename it.

'markets' is defined here:


(lint/suspicious/noRedeclare)